app/scripts/lib/channels/notifier.js

```diff
@@ -11,20 +11,56 @@ define(function (require, exports, module) {
 
   var _ = require('underscore');
   var Backbone = require('backbone');
+  var Validate = require('lib/validate');
 
   // Events that have the 'internal:' namespace should only be
   // handled by the content server. Other events may be handled
   // both externally and internally to the content server.
   var EVENTS = {
-    COMPLETE_RESET_PASSWORD_TAB_OPEN: 'fxaccounts:complete_reset_password_tab_open',
-    DELETE: 'fxaccounts:delete',
-    PROFILE_CHANGE: 'profile:change',
-    SIGNED_IN: 'internal:signed_in',
-    SIGNED_OUT: 'fxaccounts:logout'
+    COMPLETE_RESET_PASSWORD_TAB_OPEN: {
+      name: 'fxaccounts:complete_reset_password_tab_open',
+      schema: null
+    },
+    DELETE: {
+      name: 'fxaccounts:delete',
+      schema: { uid: 'String' }
+    },
+    PROFILE_CHANGE: {
+      name: 'profile:change',
+      schema: { uid: 'String' }
+    },
+    SIGNED_IN: {
+      name: 'internal:signed_in',
```

Comment:
I may be splitting hairs here, but I disagree. The term `command` there seems synonymous with the term `event` from this module, not `name`.

I only introduced `name` here so that I could distinguish between the two objects `EVENTS` and `EVENT_NAMES`. The second of those concerns itself with the strings that identify the events, not with the events themselves. Calling that object `EVENT_COMMANDS` instead would seem less clear to me, especially as some of them aren't commands in the semantic sense.

Am I making sense?